src/classes/RLLP_OppRollup_UTIL.cls

```diff
@@ -319,67 +317,54 @@ public without sharing class RLLP_OppRollup_UTIL {
+            } else {
+                multiCurrency = false;
             }
-            else
-                multiCurrency = false; 
-        }        
+        }
         return multiCurrency;
     }
-    /*********************************************************************************************************
-    * @description Converts the amount from corporate currency to the currency specified by the ISO code.
-    * @param ISO The ISO code.
-    * @param amt The amount to convert.
-    * @return Decimal The converted amount.
-    */
-    public static decimal convertFromCorporate(String ISO,Decimal amt){
-        if(currCorporate == ISO || currCorporate == null) // no need to convert
-            return amt;
-        else //return value to 2DP
-            return (amt * exchangeRates.get(ISO)).setScale(2,System.RoundingMode.UP);
+
+    /*******************************************************************************************************
+    * @description Converts an amount from the corporate currency to a given currency.
+    * @param ISO A string of the currency's iso code that we wish to convert the amount to.
+    * @param amt A decimal of the amount to convert.
+    * @return decimal The converted amount.
+    ********************************************************************************************************/
+    //do a conversion from amount in corporate currency to the currency specified by the ISO code
+    public static decimal convertFromCorporate(String ISO,Decimal amt) {
+        return convertCurrency(currCorporate, ISO, amt);
     }
-    /*********************************************************************************************************
-    * @description Converts the amount from an ISO code to the currency of the specified ISO code.
-    * @param fromISO The ISO code to convert from.
-    * @param toISO The ISO code to convert to.
-    * @param amt The amount to convert.
-    * @return Decimal The converted amount.
-    */
-    public static decimal convertCurrency(String fromISO, String toISO, Decimal amt){
-        if(fromISO == toISO) // no need to convert
-            return amt;
-        else //return value to 2DP
-            return ((amt / exchangeRates.get(fromISO)) * exchangeRates.get(toISO)).setScale(2,System.RoundingMode.UP);
+
+    /*******************************************************************************************************
+    * @description Converts an amount from any given currency to any other currency.
+    * @param fromISO A string of the currency's iso code of the amount we wish to convert.
+    * @param toISO A string of the currency's iso code that we wish to convert the amount to.
+    * @param amt A decimal of the amount to convert.
+    * @return decimal The converted amount.
+    ********************************************************************************************************/
+    public static decimal convertCurrency(String fromISO, String toISO, Decimal amt) {
```

Comment:
Is this needed or can we call UTIL_CurrencyConversion directly from wherever this is currently referenced?